packages/nextjs/lib/rules/pages-need-get-link-function-export.ts

```diff
@@ -70,12 +75,17 @@ export default creator<Options, MessageIds>({
                     return
                 }
 
+                const packageBasePath = getPackageRoot(path)
+
                 if (node.type !== "ExportNamedDeclaration") {
                     return
                 }
 
                 // no page file
-                if (!path?.includes("pages")) {
+                if (
+                    !path?.includes(packageBasePath + "/pages") &&
+                    !path?.includes(packageBasePath + "/src/pages")
+                ) {
                     return
                 }
```

Comment:
That is exactly the same package root paths check code twice - maybe we should move this up one scope